packages/e2e-test-utils/src/shopping.js

```diff
@@ -29,14 +40,14 @@ export const insertProduct = async (product) => {
     expect(page).toClick('[aria-controls="library-pane-shopping"]'),
   ]);
 
-  // This clicks on the Product dropdown and waits for the dropdown menu to open.
-  await expect(page).toClick('[aria-label="Product"]');
-  await page.waitForSelector('[aria-label="Search"]');
-
-  await page.type('[aria-label="Search"]', product);
-  await page.keyboard.press('ArrowDown');
```

Comment:
Tracked down an issue here where the search results are slightly delayed ... in turn the wrong product was getting selected.